src/locales/en.json5

```diff
@@ -1,10 +1,42 @@
 // action: This is the text for an action, for example, the text of a button.
 // field: This is the text of a form field.
 // header: This is the text of a table column header.
-
-// tab: This is the text of a navigation tab.
+// tab: This is the text of a navigation tab, which may also be shown as the page title (browser tab).
 
 {
+  // This is the text shown as the title of a page including in the browser tab.
+  "title": {
+    "resetPassword": "Reset Password",
+    "claimAccount": "Claim Account",
```

Comment:
We discussed this a little over Slack, but right now the title of the panel in `AccountClaim` is "Set Password". What do you think about reusing that text for the document title?

Reply:
I've changed the title of this AccountClaim to also say "Set Password"... and to have that text come from en.json5